portfolio/src/main/webapp/script.js

```diff
@@ -36,3 +36,29 @@ function addRandomFact() {
   funFactContainer.innerText = funFact;
   funFactContainer.style.backgroundColor = "whitesmoke";
 }
+
+function getComments() {
+  /* Clear comments before getting new comments so we don't just append comments everytime
+     the user changes how many comments they'd like to see */
+  clearComments();
```

Comment:
I think this might cause a flicker on your page since there is going to be a noticeable latency between when these comments are cleared, and when you receive the new ones from the network. Maybe move this into the response handling instead, so you clear them right before replacing them with the new ones?

Reply:
I did what I believe you suggested. Please correct me if I did not.

Reply:
I don't think I see any change here? Maybe it didn't get pushed yet? I'm just saying that you should call `clearComments()` after the response comes back. You can do it right before you call `comments.forEach(addComment)` below.

Reply:
Yes I did that, I realize I forgot to push it but it's done now.